library/net_sockets.c

```diff
@@ -465,6 +465,13 @@ int mbedtls_net_poll( mbedtls_net_context *ctx, uint32_t rw, uint32_t timeout )
     if( fd < 0 )
         return( MBEDTLS_ERR_NET_INVALID_CONTEXT );
 
+    /* A limitation of select() is that it only works with file descriptors
+     * up to FD_SETSIZE. This is a limitation of the fd_set type. Error out
```

Comment:
Again, I think `up to` is ambiguous, I think `less than` would be clearer. (Same in the comment in `recv_timeout()`.)